commit 843565c31e1ea16b7bb2dc5376d1a28884e7535a (refs/changes/55/755/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-10-20T15:37:28-07:00 (3 years ago)
    
    IMGAPI-598 images.jo and updates.jo search pages have Google Ads

diff --git a/Makefile b/Makefile
index b8cdf2e..fa2956c 100644
--- a/Makefile
+++ b/Makefile
@@ -17,7 +17,7 @@
 #
 NAME		:= imgapi
 
-DOC_FILES	 = index.md operator-guide.md search.md
+DOC_FILES	 = index.md operator-guide.md
 EXTRA_DOC_DEPS += deps/restdown-brand-remora/.git
 RESTDOWN_FLAGS   = --brand-dir=deps/restdown-brand-remora
 
diff --git a/deps/restdown-brand-remora b/deps/restdown-brand-remora
index 9945b5c..f0fd8ff 160000
--- a/deps/restdown-brand-remora
+++ b/deps/restdown-brand-remora
@@ -1 +1 @@
-Subproject commit 9945b5c9024ae6124896ceb8f6e9b5318ad516bc
+Subproject commit f0fd8ff870fa28c851e209c16cd2f90305990ddf
diff --git a/docs/search.md b/docs/search.md
deleted file mode 100644
index 4e54cf6..0000000
--- a/docs/search.md
+++ /dev/null
@@ -1,30 +0,0 @@
----
-title: Search Results
-markdown2extras: tables, code-friendly
----
-<!--
-    This Source Code Form is subject to the terms of the Mozilla Public
-    License, v. 2.0. If a copy of the MPL was not distributed with this
-    file, You can obtain one at http://mozilla.org/MPL/2.0/.
--->
-
-<!--
-    Copyright (c) 2014, Joyent, Inc.
--->
-
-# Search Results
-
-<script>
-  (function() {
-    var cx = '014195178152595988571:skaaahd6_qa';
-    var gcse = document.createElement('script'); gcse.type = 'text/javascript'; gcse.async = true;
-    gcse.src = (document.location.protocol == 'https:' ? 'https:' : 'http:') + '//www.google.com/cse/cse.js?cx=' + cx;
-    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(gcse, s);
-  })();
-</script>
-<div class="row top">
-    <? snippet('devsidebar') ?>
-    <div class="span8">
-        <gcse:searchresults-only>
-    </div>
-</div>
